internal/client/resource_group.go

```diff
@@ -0,0 +1,132 @@
+package client
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+const resourceGroupBasePath = "/api/resource_groups"
+
+type ResourceGroup struct {
+	ID          int64   `json:"id"`
+	Name        string  `json:"name"`
+	Description *string `json:"description"`
+	CreatedAt   string  `json:"created_at"`
```

Comment:
those parameters is necessary?  If it is not managed by terraform, it is not necessary.

Reply:
I have just moved the original struct. (There was mistake in another part, though.)
but I re-considered that there is no need to move it 
because it is not used in resource_group as a resource.

Reply:
ResourceGroup is just moved and not used in this modification.
So I'll undo what had been moved in the definition.